lib/params/RenderParams.cpp

```diff
@@ -51,6 +51,13 @@ const string RenderParams::_RefinementLevelTag = "RefinementLevel";
 const string RenderParams::_transferFunctionsTag = "MapperFunctions";
 const string RenderParams::_stretchFactorsTag = "StretchFactors";
 const string RenderParams::_currentTimestepTag = "CurrentTimestep";
+const string RenderParams::XOriginTag = "XOrigin";
```

Comment:
Sorry to nit pick on this, but "origin" and "rotation"  are perhaps too general for something that is intended to describe position and orientation of a specific thing, a cutting or slice plane. Lots of things could have an origin or a rotation. How about something like

XSlicePlaneOrigin, XSlicePlaneRotation, etc?

Reply:
No problem.  Origin and Rotation parameter tags are updated.

IE:

XOriginTag = XSlicePlaneOriginTag
XRotationTag = XSlicePlaneRotationTag